pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -599,9 +599,11 @@ private static File convertURIToSegmentLocation(String segmentLocation) {
    * @param rawTableName Raw table name
    * @param committingSegmentNameStr Committing segment name
    * @param nextOffset The offset with which the next segment should start.
+   * @param memoryUsed The memory used by committing segment
```

Comment:
pls add comments here as to what the value will be if the client did not set it at all.

Reply:
Also, add the units. I am assuming this is in Bytes.

Reply:
+1, very good point.

Reply:
Added units. Also added comments about behaviour when this is not set by client